sv/git.go

```diff
@@ -15,8 +15,8 @@ import (
 )
 
 const (
-	logSeparator = "##"
-	endLine      = "~~"
+	logSeparator = "###"
```

Comment:
Yes. We use Markdown for description of huge PRs. When header on level (##) have been used this confuses sv4git because of the lineSeparator. Maybe a whole new char for separating lines might also be helpful. Currently this strike again if Markdown header on level 3 will be used.

Reply:
oh, got it.. probably it will be better to change to another character..